scripts/scil_tracking_local_dev.py

```diff
@@ -150,6 +150,13 @@ def _build_arg_parser():
                           "with -nt 1,000,000, \nyou can create tractogram_2 "
                           "with \n--skip 1,000,000.")
 
+    track_g.add_argument('--hurdle_mask', default=None,
+                         help='Hurdle mask (.nii.gz).\n'
+                        'Hurdle rracking will start within this mask.')
+    track_g.add_argument('--hurdle_method', default='continue',
+                        choices=['continue'],
+                        help="Method to solve hurdles [%(default)s]")
```

Comment:
I know we don't care about that for now but shouldn't this default to `none` to keep the same default behaviour as before?

Reply:
The RAP tracking wasn't used if the RAP_mask is None. For clarity, I added None, by default.